website/api/controllers/entrance/signup.js

```diff
@@ -36,7 +36,7 @@ the account verification message.)`,
     },
 
     organization: {
-      required: true,
+      defaultsTo: '',
```

Comment:
`defaultsTo: ''` is the implicit default value for `type: 'string'`, so this can be removed.

Reply:
I'll tweak this in a commit to save a roundtrip here.

Reply:
Actually @eashaw, my reasoning for the above is faulty.

While we can still remove `defaultsTo: ''` here, the reason is more nuanced.  While you never need `required: false`, no matter what, specifically in the case of validating inputs, the value will be `undefined` for an optional input that ends up being unspecified at runtime, regardless of its type.

The reason why this `defaultsTo: ''` can be omitted here is that `organization` is passed through to `User.create()`, where `organization` is an optional attribute.

So by passing in `undefined`, it leaves it up to the ORM, which then, because of how `type:'string'` works in Waterline, is actually persisted as empty string (`''`) automatically.  And that's what will come back any time it is queried.

Hope that helps- sorry for any confusion my incomplete explanation above might have caused.

Reply:
See https://github.com/fleetdm/fleet/pull/6380/commits/2665abef1dbb471086cc5888be01f805ef456472#diff-9c6a73c96fdd057f48cfb92925ac8cbf02462ac4889c1ee6dd5bc88cec546465L39 and https://github.com/fleetdm/fleet/pull/6380/commits/e150047bb9adc2ad237ae02994ef3ad0528ed231